retrofit-converters/wire/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java-library'
+apply plugin: 'com.vanniktech.maven.publish'
+
+dependencies {
+  api project(':retrofit')
+  api(deps.wireRuntime) {
+    // Make sure OkHttp's transitive version wins (itself transitive from Retrofit).
```

Comment:
Why’s this necessary? I’d expect the highest number would be better policy

Reply:
Unfortunately Maven isn't that smart. Okio from Retrofit is a 3rd order dependency and Okio from Wire is a 2nd order dep. So Maven selects the Wire one.

Gradle consumers don't have this problem.